neat_ml/yaml_helper/yaml_helper.py

```diff
@@ -426,25 +472,42 @@ def make_upload_args(self) -> dict:
     #
     # applying trained model to fresh data for predictions
     #
-    def do_apply_classifier(self):
+    def do_apply_classifier(self) -> list:
+        """Apply classifier.
+
+        :return: List of classifiers to be applied.
+        """
         return "ApplyTrainedModelsContainer" in self.yaml
```

Comment:
@caufieldjh , Mypy seems to complain about this and I see the reason why. The function is expected to return a `list` but this statement would result in a `Boolean`.

Reply:
I changed it to `return self.yaml["ApplyTrainedModelsContainer"]` , fyi.

Reply:
I dug a little deeper and this is what I found:

This code breaks while running `test_visuals.py`. The `self.yaml` object looks like this:

```
{
   "Target":{
      "target_path":"tests/resources/test_output_data_dir/"
   },
   "GraphDataConfiguration":{
      "graph":{
         "directed":false,
         "node_path":"tests/resources/test_graphs/test_small_nodes.tsv",
         "edge_path":"tests/resources/test_graphs/test_small_edges.tsv",
         "verbose":true,
         "nodes_column":"id",
         "node_list_node_types_column":"category",
         "default_node_type":"biolink:NamedThing",
         "sources_column":"subject",
         "destinations_column":"object",
         "default_edge_type":"biolink:related_to"
      }
   },
   "EmbeddingsConfig":{
      "filename":"test_embeddings.csv",
      "node_embeddings_params":{
         "method_name":"SkipGram",
         "walk_length":10,
         "batch_size":128,
         "window_size":4,
         "return_weight":1.0,
         "explore_weight":1.0,
         "iterations":5
      },
      "tsne_filename":"test_tsne.png"
   }
}
```

There doesn't seem to be a "ApplyTrainedModelsContainer" key anywhere. I don't have enough information to pin-point what needs to change. We could go through it when we get a chance.

Reply:
Yeah, that should just be a bool since it should be False if there aren't any models to apply, according to the config yaml. I fixed it.